fish_tealdeer

```diff
@@ -3,19 +3,20 @@
 # https://github.com/dbrgn/tealdeer/
 #
 
-complete -c tldr -s h -l help        -d 'Print the help message.' -f
-complete -c tldr -s v -l version     -d 'Show version information.' -f
-complete -c tldr -s l -l list        -d 'List all commands in the cache.' -f
-complete -c tldr -s f -l render      -d 'Render a specific markdown file.' -r
-complete -c tldr -s p -l platform    -d 'Override the operating system.' -xa 'linux macos sunos windows'
-complete -c tldr -s u -l update      -d 'Update the local cache.' -f
-complete -c tldr -s c -l clear-cache -d 'Clear the local cache.' -f
-complete -c tldr -s p -l pager       -d 'Use a pager to page output.' -f
```

Comment:
This `-s p` is a mistake right?

Reply:
Yes, well spotted, the short version of that should be removed. A fix in a separate PR (once this is merged) would be welcome 🙂

Reply:
I opened a PR here: https://github.com/dbrgn/tealdeer/pull/262